activities/workshops/index.md

```diff
@@ -8,5 +8,6 @@ Here is a list of workshops developed or used by the Foundation for Public Code:
 
 * [Governance exercise](../supporting-codebase-governance/exercise/index.md)
 * [Governance game](../supporting-codebase-governance/game/index.md)
+* [Mapping business models enabled by codebase](mapping-business-models.md)
```

Comment:
* fix the link in the index.md of the workshop folder pointing to the mapping-business-models folder (link is broken)

Reply:
Done!